Makefile

```diff
@@ -48,9 +48,9 @@ populate: create_db  ## Build and run containers
 .PHONY: test
 test: start  ## Run tests
 	if [ -z "$(name)" ]; then \
-		FLASK_ENV=testing docker-compose run --rm web pytest --doctest-modules -n auto --dist=loadfile -v OpenOversight/tests/; \
+		FLASK_ENV=testing docker-compose run --rm web pytest --cov=OpenOversight --cov-report xml:OpenOversight/tests/coverage.xml --doctest-modules -n auto --dist=loadfile -v OpenOversight/tests/; \
```

Comment:
These are only used for local testing, but it seems fine to generate coverage reports here as well